lib/PuppeteerSharp/EvaluateExceptionStackTrace.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System.Collections.Generic;
+
+namespace PuppeteerSharp
+{
+    public class EvaluateExceptionStackTrace
+    {
+        public IEnumerable<EvaluationExceptionCallFrame> CallFrames { get; internal set; }
```

Comment:
is this missing a `JsonProperty` attribute?

this is only initialized from DeSerialization so it might as well be an Array, not sure about this